From 3d292ae5c79deb360f3bd9dc4192bf4613fa29ce Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Tue, 5 Mar 2013 12:02:20 +0800
Subject: [PATCH] rk3168/rk3188: board enable CONFIG_UHID in defconfig

Enable CONFIG_UHID to allow userspace to inject HID packets.
---
 arch/arm/configs/rk3168_86v_codec_defconfig    | 1 +
 arch/arm/configs/rk3168_86v_defconfig          | 1 +
 arch/arm/configs/rk3168_86v_old_defconfig      | 1 +
 arch/arm/configs/rk3168_LR097_defconfig        | 1 +
 arch/arm/configs/rk3168_ds1006h_defconfig      | 1 +
 arch/arm/configs/rk3168_tb_defconfig           | 1 +
 arch/arm/configs/rk3188_LR097_defconfig        | 1 +
 arch/arm/configs/rk3188_ds1006h_defconfig      | 1 +
 arch/arm/configs/rk3188_ds1006h_v1_0_defconfig | 1 +
 arch/arm/configs/rk3188_tb_defconfig           | 1 +
 10 files changed, 10 insertions(+)

diff --git a/arch/arm/configs/rk3168_86v_codec_defconfig b/arch/arm/configs/rk3168_86v_codec_defconfig
index af682af796f2..25d6d06b7f8f 100644
--- a/arch/arm/configs/rk3168_86v_codec_defconfig
+++ b/arch/arm/configs/rk3168_86v_codec_defconfig
@@ -309,6 +309,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RT5631=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
diff --git a/arch/arm/configs/rk3168_86v_defconfig b/arch/arm/configs/rk3168_86v_defconfig
index 41f3c986283c..98b71b27e416 100644
--- a/arch/arm/configs/rk3168_86v_defconfig
+++ b/arch/arm/configs/rk3168_86v_defconfig
@@ -313,6 +313,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RK610=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
diff --git a/arch/arm/configs/rk3168_86v_old_defconfig b/arch/arm/configs/rk3168_86v_old_defconfig
index cbeeb64e1253..1ac1d7f40d7b 100644
--- a/arch/arm/configs/rk3168_86v_old_defconfig
+++ b/arch/arm/configs/rk3168_86v_old_defconfig
@@ -314,6 +314,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RK610=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
diff --git a/arch/arm/configs/rk3168_LR097_defconfig b/arch/arm/configs/rk3168_LR097_defconfig
index c42e038dc74f..49e47041a564 100644
--- a/arch/arm/configs/rk3168_LR097_defconfig
+++ b/arch/arm/configs/rk3168_LR097_defconfig
@@ -315,6 +315,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RK610=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
diff --git a/arch/arm/configs/rk3168_ds1006h_defconfig b/arch/arm/configs/rk3168_ds1006h_defconfig
index 52086a4e3de9..38db4ca62a6c 100644
--- a/arch/arm/configs/rk3168_ds1006h_defconfig
+++ b/arch/arm/configs/rk3168_ds1006h_defconfig
@@ -328,6 +328,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RK610=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
diff --git a/arch/arm/configs/rk3168_tb_defconfig b/arch/arm/configs/rk3168_tb_defconfig
index 82fbc4f8e313..153387c24f2b 100755
--- a/arch/arm/configs/rk3168_tb_defconfig
+++ b/arch/arm/configs/rk3168_tb_defconfig
@@ -323,6 +323,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RT5631=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
diff --git a/arch/arm/configs/rk3188_LR097_defconfig b/arch/arm/configs/rk3188_LR097_defconfig
index dae5d7ee30ca..c5878194b85c 100644
--- a/arch/arm/configs/rk3188_LR097_defconfig
+++ b/arch/arm/configs/rk3188_LR097_defconfig
@@ -321,6 +321,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RT5631=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
diff --git a/arch/arm/configs/rk3188_ds1006h_defconfig b/arch/arm/configs/rk3188_ds1006h_defconfig
index a41515acfe90..855846fcd2dc 100644
--- a/arch/arm/configs/rk3188_ds1006h_defconfig
+++ b/arch/arm/configs/rk3188_ds1006h_defconfig
@@ -329,6 +329,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RK610=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
diff --git a/arch/arm/configs/rk3188_ds1006h_v1_0_defconfig b/arch/arm/configs/rk3188_ds1006h_v1_0_defconfig
index 70110426f0f7..d5e2ddfc83f9 100644
--- a/arch/arm/configs/rk3188_ds1006h_v1_0_defconfig
+++ b/arch/arm/configs/rk3188_ds1006h_v1_0_defconfig
@@ -326,6 +326,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RK610=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
diff --git a/arch/arm/configs/rk3188_tb_defconfig b/arch/arm/configs/rk3188_tb_defconfig
index 351ad1a35dff..a85fb25275b8 100644
--- a/arch/arm/configs/rk3188_tb_defconfig
+++ b/arch/arm/configs/rk3188_tb_defconfig
@@ -322,6 +322,7 @@ CONFIG_SND_RK29_SOC_I2S_2CH=y
 CONFIG_SND_I2S_DMA_EVENT_STATIC=y
 CONFIG_SND_RK29_SOC_RT5631=y
 CONFIG_SND_RK29_CODEC_SOC_SLAVE=y
+CONFIG_UHID=y
 CONFIG_HID_A4TECH=y
 CONFIG_HID_ACRUX=y
 CONFIG_HID_ACRUX_FF=y
-- 
2.35.3

